resolwe_bio/processes/import_data/bam.yml

```diff
@@ -180,56 +180,6 @@
           re-error "BAI should have the same name as BAM with .bai extension"
         fi
 
-- slug: upload-bam-primary
```

Comment:
Why is this not needed anymore?

Reply:
because `bam-split` process data object is now of a `data:alignment:bam:primary` type so it can be directly used in workflows (process spawned processes can't be used).